releasenotes/notes/add_ControlledGate_class-010c5c90f8f95e78.yaml

```
@@ -0,0 +1,27 @@
+---
+features:
+  - |
+    Add ``ControlledGate`` class for representing controlled
+    gates. Controlled gate instances are created with the
+    ``control(n)`` method of ``Gate`` objects where ``n`` represents
+    the number of controls. The control qubits come before the
+    controlled qubits in the new gate. For example::
+
```

Can we add the imports here too? I think it's just that most people don't necessarily where to get `HGate`. Something like::

```suggestion

      from qiskit import QuantumCircuit
      from qiskit.extensions import HGate

```